### PR TITLE
Qe work michael kolodner: QE App with QE specific record pages, data, and more

### PR DESCRIPTION
**Pull request by Szandor72**
Follow-Up for #1371 

# Critical Changes

# Changes
## QE
- added more QE config data, i.e.
- QE App, Record Pages, Permset, tiny Apex Helper
- clones of `LookUpChild__c` and `LookupParent__c` to avoid dependencies of QA work on package internals 
- adjusted cumulusci.yml

## Houesekeeping
- updated .forceignore ON PULL for files we don't care about or cannot / should not package
- updated default folder structure so that pulling with dx doesn't create additional folders anymore
- removed unnecessary files 

# Issues Closed

### CODE REVIEW

File: datasets/qa/mapping.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ Insert Account:
   - Name
   - NumberOfLocations__c
   - SLAExpirationDate__c
+  - Description
```
Comment (Szandor72) [inline datasets/qa/mapping.yml:7]: @mkolodner Would you remove those from the mapping and datasets if present?

Comment (mkolodner) [reply datasets/qa/mapping.yml:7]: @Szandor72  Removed and new commit published. Not sure how to put that commit into this PR....?

Comment (Szandor72) [reply datasets/qa/mapping.yml:5]: I moved the commit for you. 

In order to work on this PR, you need to check out the new branch `feature/QE-Work-Michael-Kolodner`

File: unpackaged/config/qa/classes/QAHelper.cls [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
```
Comment (davidmreed) [inline unpackaged/config/qa/classes/QAHelper.cls:1]: Another option (at the team's discretion) is to run this as Anonymous Apex so that the class is not deployed into the org. Having the class deployed can sometimes cause confusion.

File: unpackaged/config/qa/objects/Account/fields/SLAExpirationDate__c.field-meta.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (davidmreed) [inline unpackaged/config/qa/objects/Account/fields/SLAExpirationDate__c.field-meta.xml:1]: This field is part of standard scratch org metadata - do we intend to use/capture it?

File: unpackaged/config/qa/objects/Account/fields/NumberOfLocations__c.field-meta.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (davidmreed) [inline unpackaged/config/qa/objects/Account/fields/NumberOfLocations__c.field-meta.xml:1]: This field is part of standard scratch org metadata - do we intend to use/capture it?

File: dlrs/default/lwc/jsconfig.json
Comment (davidmreed) [inline dlrs/default/lwc/jsconfig.json:1]: Is this change intentional? It does not appear related to the rest of the PR.

File: dlrs/default/main/default/aura/.eslintrc.json
Comment (davidmreed) [inline dlrs/default/main/default/aura/.eslintrc.json:1]: Is this change intentional? It does not appear related to the rest of the PR.

File: dlrs/default/main/default/lwc/.eslintrc.json
Comment (davidmreed) [inline dlrs/default/main/default/lwc/.eslintrc.json:1]: Is this change intentional? It does not appear related to the rest of the PR.

**Review by davidmreed — DISMISSED** [review]

**Review by davidmreed — APPROVED** [review]
